src/components/navigation/navbar/Navbar.tsx

```diff
@@ -12,7 +12,11 @@ export const Navbar = () => {
   let location = useLocation();
 
   useEffect(() => {
-    location.pathname !== "/" ? setToogle(true) : setToogle(false);
+    if (location.pathname === "/") {
+      setToogle(false);
+    } else if (location.pathname === "/favorites") {
+      setToogle(true);
+    }
```

Comment:
These flags will quickly get out of hand with all the navigation items. Use an enum of values and a selected value. If the current item is a selected one, apply additional styling.

Reply:
I'm not 100% sure what exactly you want to say, but I think I have an idea how to solve it...